### ISSUE TITLE
[Accessibility]: LineChart - Tooltip information provided for the curve chart is not announced by SR

### ISSUE THREAD

**Issue by amortiz** [carbon-design-system/carbon-charts#2020]
### Application/Team

IBM Infrastructure

### Environment

Assistive technology used to verify

### What happened? What did you expect to happen instead?

AVT issue for LineChart - tooltip, see report below:

Component: Tool tip provided Chart Graphical Representaion

Issue description: Tool tip information provided for the curve chart is not announced by SR and Value varies on each hover on the graph

Recommended fix: The spin box should be provided for the screen reader usder or keyboard user to under stand the content provided in the curve graph.

User impact: If screen reader doesn't announce the data, then it will be difficult for the screen reader users to get the information.

WCAG Mapping: 1.1.1 Non-Text Content

Level of Compliance: A

Severity: High

Test tool: JAWS_Chrome

### What WCAG 2.1 checkpoint does the issue violate?

WCAG Mapping: 1.1.1 Non-Text Content

### Version

@carbon/charts-react@1.23.17

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P3
